src/Adapters/TraceListener.Shared/ApplicationInsightsTraceListener.cs

```diff
@@ -47,6 +47,16 @@ public ApplicationInsightsTraceListener(string instrumentationKey)
             this.TelemetryClient.Context.GetInternalContext().SdkVersion = SdkVersionUtils.GetSdkVersion("sd:");
         }
 
+        /// <summary>
+        /// Initializes a new instance of the ApplicationInsightsTraceListener class, specifying a TelemetryClient to use
+        /// </summary>
+        /// <param name="telemetryClient">TelemetryClient to use for tracing</param>
+        public ApplicationInsightsTraceListener(TelemetryClient telemetryClient)
```

Comment:
Hmm, it's complaining that the TelemetryConfiguration constructor parameter is not CLS compliant. I assume that's important to you guys :) Thoughts?